manifests/plugin/remote_execution/ssh.pp

```diff
@@ -49,6 +52,18 @@
     template_path => 'foreman_proxy/plugin/remote_execution_ssh.yml.erb',
   }
 
+  if $ssh_kerberos_auth {
+    if $::osfamily == 'RedHat' and $::operatingsystem != 'Fedora' {
```

Comment:
This is an optional dependency for the `smart_proxy_dynflow_core` service which runs in SCL